.prettierignore

```diff
@@ -0,0 +1 @@
+package.json
```

Comment:
When I would modify package.json and save in my IDE, Prettier would format the whole file. It doesn't seem like Prettier is set up to do that in this repo, so I thought ignoring that file would prevent this / keep the package.json diff cleaner in this PR. This alright?